docs/gmsl.html

```diff
@@ -22,7 +22,7 @@ <h2>Using GMSL</h2>
 you have the right version of <span style="font-family: monospace;">gmsl</span>
 use the <span style="font-family: monospace;">gmsl_compatible</span>
 function (see
-below). The current version is <span style="font-family: monospace;">1 1 7</span>.<br>
+below). The current version is <span style="font-family: monospace;">1 1 6</span>.<br>
```

Comment:
1 1 8

Reply:
Huh. Must have been a merge error when I edited the commit.